anonymizer/OpenDiffix.Service/Program.fs

```diff
@@ -35,17 +36,32 @@ let csvFormat value =
   | String string -> quoteString string
   | _ -> Value.toString value
 
-let csvFormatter result =
+let csvFormatter suppressedAnonCount result =
```

Comment:
Can't the star be injected into the `result`?

Reply:
Or some `extraRows` parameter.

Reply:
yeah, I am not happy with what is here now. At first, I didn't want to complicate the code too much with injection or some more elaborate handling... this is ugly but at least straightforward. I'll work on improving this.